bin/einhorn

```diff
@@ -310,6 +310,10 @@ if true # $0 == __FILE__
       Einhorn::State.signal_timeout = Integer(t)
     end
 
+    opts.on('-r', '--resignal-timeout', 'Only applies if --signal-timeout is set. Resignal all children instead of just the ones most recently terminated.') do
+      Einhorn::State.config[:resignal_timeout] = true
```

Comment:
What's the difference between this and `Einhorn::State.signal_timeout`, used below?

Reply:
Are they the same? Is `Einhorn::State.signal_timeout` some vestige of something?

Reply:
`--signal-timeout N` is the original flag that will send a `KILL` to a process if it is still running N seconds after an upgrade was issued. `resignal-timeout` is a new flag that opts in to the new behaviour in this PR.